.azure/pipelines/site-extensions.yml

```diff
@@ -5,6 +5,8 @@ trigger:
 
 phases:
 - phase: SiteExtensions
+  pool:
+    name: dotnet-external-temp
```

Comment:
This needs to vary based on which project we build in. See https://github.com/dotnet/arcade/blob/master/Documentation/ChoosingAMachinePool.md and

https://github.com/aspnet/AspNetCore/blob/ef1fc9e5c74b93ae45b8f4b36a4af0ea8bce9b43/.azure/pipelines/jobs/default-build.yml#L89-L92

Reply:
This is never built in public.

Reply:
Fixed the name,